README.md

```diff
@@ -77,6 +78,7 @@ Lightning Terminal is backwards compatible with `lnd` back to version v0.11.1-be
 
 | LiT              | LND          | Loop        | Faraday      | Pool          |
 | ---------------- | ------------ | ----------- | ------------ |---------------|
+| **v0.3.5-alpha** | v0.12.0-beta | v0.11.2-beta | v0.2.3-alpha | v0.4.2-alpha |
```

Comment:
Why v0.3.5? AFAIK we want the next release to be v0.4.0 and include the pool ui.

Reply:
I wasn't sure if the UI would be ready together with lnd v0.12. But in that case it makes sense to update to `v0.4.0` in the PR that adds the UI. Going to remove this diff then.

Reply:
Cool 👍